src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -1,682 +1,1148 @@
+              size="md"
+              (click)="deleteCrushRule()"
+            >
+              <svg
+                cdsIcon="trash-can"
+                size="32"
+                class="cds--btn__icon"
+              >
+              </svg>
+            </cds-icon-button>
+            }
+          </div>
+          }
+        </div>
+        }
+
+        <div
+          class="form-item"
+          cdsRow
+        >
+          <div
+            cdsCol
+          >
+            @if (data.crushInfo && form.getValue('crushRule')) {
+            <span
+              class="form-text text-muted"
+              id="crush-info-block"
+            >
+              <cds-tabs
+                [type]="'contained'"
+                [followFocus]="true"
+                [isNavigation]="false"
+                [cacheActive]="true"
+              >
+                <cds-tab
+                  heading="Crush rule"
+                  i18n-heading
+                  [tabContent]="crushRuleTpl"
+                >
+                </cds-tab>
+                <cds-tab
+                  heading="Crush steps"
+                  i18n-heading
+                  [tabContent]="crushStepsTpl"
+                >
+                </cds-tab>
+                <cds-tab
+                  heading="Used by pools"
+                  i18n-heading
+                  [tabContent]="usedPoolTpl"
+                >
+                </cds-tab>
+              </cds-tabs>
+              <ng-template #crushRuleTpl>
+                <cd-table-key-value
+                  [renderObjects]="false"
+                  [hideKeys]="['steps', 'type', 'rule_name']"
+                  [data]="this.selectedCrushRule"
+                  [autoReload]="false"
+                >
+                </cd-table-key-value>
               </ng-template>
+              <ng-template #crushStepsTpl>
+                <ol>
+                  @for (step of this.selectedCrushRule?.steps; track $index) {
+                  <li>
+                    {{ describeCrushStep(step) }}
+                  </li>
+                  }
+                </ol>
+              </ng-template>
+              <ng-template #usedPoolTpl>
+                <ng-template #ruleIsNotUsed>
+                  <span i18n>
+                    Rule is not in use.
+                  </span>
+                </ng-template>
+                @if (crushUsage) {
+                <ul>
+                  @for (pool of crushUsage; track pool) {
+                  <li>
+                    {{ pool }}
+                  </li>
+                  }
+                </ul>
+                } @else {
+                <ng-container *ngTemplateOutlet="ruleIsNotUsed"></ng-container>
+                }
+              </ng-template>
+            </span>
+            }
+          </div>
+        </div>
+      </div>
+      }
+      <!-- Compression -->
+      @if (info?.is_all_bluestore) {
+      <div formGroupName="compression">
+        <legend i18n>
```

Comment:
let's use the `cd-header` class for headings which already has the carbon header classes applied and the font looks more closer to carbon that way. 

Same with all the places that we use legend for header.
```suggestion
        <legend class="cd-header" i18n>
```

Reply:
Done